cli/src/main/java/com/devonfw/tools/ide/commandlet/FileExtractor/ExctractorFileType.java

```diff
@@ -0,0 +1,5 @@
+package com.devonfw.tools.ide.commandlet.FileExtractor;
+
+enum ExtractorFileType {
```

Comment:
Thanks for adding these, some javadoc would be nice though.

Reply:
Done